docs/core_concepts/datasource.rst

```diff
@@ -23,6 +21,9 @@ a SqlAlchemyDataset corresponding to that batch of data and ready for validation
 Since opinionated DAG managers such as airflow, dbt, prefect.io, dagster can also act as datasources
 and/or generators for a more generic datasource.
 
+When adding custom expectations by subclassing an existing DataAsset type, use the data_asset_type parameter
```

Comment:
Minor nit: `data_asset.type`, right? As opposed to `data_asset_type`...?

Reply:
No, it is data_asset_type:
```
datasources:
  my_datasource:
    class_name: PandasDatasource
    data_asset_type:
      module_name: custom_pandas_datasource
      class_name: CustomPandasDatasource
    generators:
      default:
        type: subdir_reader
        base_directory: /data
```